src/main/java/core/basesyntax/dao/StorageDao.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.dao;
+
+public interface StorageDao {

```

Comment:
Lets add method updateData in cese you editing existed one. This is how Dao should work

Reply:
p.s. you can read about CRUD